modules/__tests__/radium-test.js

```diff
@@ -0,0 +1,158 @@
+var Radium = require('index.js');
+var React = require('react/addons');
+
+var {Component} = React;
+var TestUtils = React.addons.TestUtils;
+
+// NOTE: Many of these tests use mouseDown and active styles. For some reason,
```

Comment:
This, maybe? https://github.com/facebook/react/issues/1297

Current workaround is `TestUtils.SimulateNative.mouseOver`, if so. Will be fixed in 0.14.